Inventory.cpp

```diff
@@ -51,41 +51,42 @@ void Inventory::displayInventory() const
 	for (int i = 0; i < inventorySpace; i++)
 	{
 		std::cout << i + 1 << ". " << itemInSlot[i];
-		if (quantityInSlot[i] > 0){std::cout << " X" << quantityInSlot[i] << std::endl;}
-		else { std::cout << std::endl; }
+
+		if (quantityInSlot[i] > 0) {
+			std::cout << " X" << quantityInSlot[i] << std::endl;
+		} else {
+			std::cout << std::endl;
+		}
 	}
 }
 
 //Add an item to inventory
 void Inventory::addToInventory(std::string invItem)
 {
-	for (int i = 0; i < inventorySpace; i++)
-	{
-		if ((invItem == itemInSlot[i]) && quantityInSlot[i] < 2)
-		{
+	for (int i = 0; i < inventorySpace; i++) {
```

Comment:
The common style is to put the opening brace on the same line as the statement that necessitates it. It's not uncommon for some styles to use the style you're using right now, but I think it's the edge-case/minority. This style is more commonly used in open source code I see online, and makes better use of the lines. IMO it's best to adopt the common styles and keep in mind the styles of the stuff you're working on, and adapt to them on a case-by-case basis.